scripts/omg.yml

```diff
@@ -27,4 +27,6 @@ runcmd:
     admin_console_install: False
     admin_console_enabled: False
     EOF
-  - curl iiab.io/risky.txt | bash &
+  - curl iiab.io/risky.txt | bash
```

Comment:
The above removal of `&` would change Step 3. and 4. of the Install instructions:

https://github.com/iiab/calibre-web/wiki#wrench-installation

Perhaps that's fine as a short-term testing hack?

(But probably doesn't make sense long-term!)

Reply:
I thought keeping it would prevent the last command to run when the previous is finished. If the sequence is guaranteed, we can keep it.

Reply:
> I thought keeping it would prevent the last command to run when the previous is finished. If the sequence is guaranteed, we can keep it.

Understood:

While there are aspects of this PR and PR #50 that would appear to be temporary hacks / interim scaffolding, that may be a Good Thing if the purpose here is in fact TDD.

(Until an even better approach is ideally found in coming days!?)

Reply:
Done.

Reply:
Thanks @deldesir for having cleared this up. ✅

Now that [scripts/omg.yml](https://github.com/iiab/calibre-web/blob/master/scripts/omg.yml) is no longer relevant to this PR.